docs/transaction-building.md

```diff
@@ -0,0 +1,220 @@
+
+In the following example, we are minting a token named `TestToken` and sending it to `9gNvAv97W71Wm33GoXgSQBFJxinFubKvE6wh2dEhFTSgYEe783j` address.
+
+<!-- prettier-ignore -->
+```ts
+new TransactionBuilder(creationHeight)
+  .from(inputs)
+  .to(
+    new OutputBuilder("1000000", "9gNvAv97W71Wm33GoXgSQBFJxinFubKvE6wh2dEhFTSgYEe783j")
+      .mintToken({ // [!code focus]
+        amount: "100", // the amount of tokens being minted without decimals // [!code focus]
+        name: "TestToken", // the name of the token // [!code focus]
+        decimals: 2, // the number of decimals  // [!code focus]
+        description: "This is a test token minted with Fleet SDK" // [!code focus]
+      }) // [!code focus]
+  );
+```
+
+:::warning
+Only one token can be minted per transaction.
+:::
+
+## Step. 5: Set the change address
+
+Often you do not use all the funds included in the input boxes, so you need to provide an address to receive these funds back. It's the so-called UTxO change.
+
+To do so, you must call the `sendChangeTo()` method and pass an address as the only parameter.
+
+In the following example, we are sending the change to `9gNvAv97W71Wm33GoXgSQBFJxinFubKvE6wh2dEhFTSgYEe783j` address.
+
+<!-- prettier-ignore -->
+```ts
+new TransactionBuilder(creationHeight)
+  // ...
+  .sendChangeTo("9gNvAv97W71Wm33GoXgSQBFJxinFubKvE6wh2dEhFTSgYEe783j"); // [!code focus]
+```
+
+## Step. 6: Set the fee amount
+
+Each transaction on Ergo needs to have a mining fee box, you can use the `payFee()` method to specify the fee amount in `nanoergs`.
+
+<!-- prettier-ignore -->
+```ts
+new TransactionBuilder(creationHeight)
+  // ...
+  .payFee("1100000"); // [!code focus]
+```
+
+Alternatively, you can use the `payMinFee()` method to add the min recommended miner fee amount, which is `1100000 nanoergs` or `0.0011 ERG`.
+
+<!-- prettier-ignore -->
+```ts
+new TransactionBuilder(creationHeight)
+  // ...
+  .payMinFee(); // [!code focus]
+```
+
+## Step. 7: Build
+
+Now that all pieces are put together, it's time to build the transaction and obtain the **Unsigned Transaction** object. For that you can use the `build()` method.
+
+The `build()` method must be the last called method on the `TransactionBuilder` as it will do all necessary validations, input selection, change calculations and return an unsigned transaction object which can be signed by any Ergo signing tool.
+
+<!-- prettier-ignore -->
+```ts
+new TransactionBuilder(creationHeight)
+   // ...
+  .build(); // [!code focus]
+```
+
+By default, the `build()` method will return a default unsigned transaction object, if you want to send it to be signed by the user's wallet through [dApp Connector protocol](./wallet-interaction.md), you should include "EIP-12" as a parameter in the `build()` method so it will include additional information required by the protocol.
+
+<!-- prettier-ignore -->
+```ts
+new TransactionBuilder(creationHeight)
+   // ...
+  .build("EIP-12"); // [!code focus]
+```
```

Comment:
Probably should either have a link to the total built transaction or have it on the end as an appendix, so someone can copy and paste the whole transaction rather than need to piece it together. This tutorial assumes someone is following a step-by-step process. If this were me, I'd want to copy the whole, read this to understand it, make modifications, then run it to see if I am using it right using a test wallet. I imagine that is probably the norm.

Reply:
Makes very sense, will add.